scripts/kickstart_generator.rb

```diff
@@ -43,23 +43,6 @@ def gen_file_path(file)
 
     private
 
-    def write_config(result)
-      file = @ks_gen_base.join("base-#{@target}.cfg")
-      $log.info("Writing kickstart in config format: #{file}") if $log
-      File.write(file, result)
-    end
-
-    def write_json(result)
```

Comment:
See above, we should really try to get this feature working with the existing json/config formats and determine if we still need both of those formats in a future PR.